pkg/model/resource/resource.go

```diff
@@ -69,7 +69,13 @@ func wrapKey(key string) string {
 func (r Resource) Replacer() *strings.Replacer {
 	var replacements []string
 
-	replacements = append(replacements, wrapKey("group"), r.Group)
+	// If group is empty replace the path with core.
+	group := r.Group
+	if group == "" {
+		group = "core"
+	}
```

Comment:
I do not think that we should replace the empty "" group by the core. I think we need be able to make the tool works with empty groups.

Reply:
This has been done to support the multi group with the empty group specified in this [comment](https://github.com/kubernetes-sigs/kubebuilder/issues/1404#issuecomment-594504374). The empty group will only have path in core directory, the PROJECT file or any configuration files will not have any group.

Reply:
@camilamacedo86 To support empty group name in multi groups without replacing the directory paths as "core". The path for empty group types.go will be "api/version/types.go" instead of "api/group/version/types.go" and same goes for the controller. Kindly suggest Is this an appropriate way to handle this?

Reply:
IMO the problem here is that we are defining the resource group as core. So, it goes to PROJECT file and etc. It will no only about the paths and imports generated. 

The tool has 2 layouts:

**default** (not support multigroup)  

> project/api/v1/<kind>_types.go 
> project/controllers/<kind>_controller.go 

Which in this case shows that would keep the same. 

However, we have the multigroup layout: 

> project/api/<version>/<group>/<kind>_types.go 
> project/controllers/<group>/<kind>_controller.go 

Then, if it is an empty group we could check if would be possible have 

> project/api/v1/<group>/<kind>_types.go 
> project/controllers/<group>/<kind>_controller.go 

AND

> project/api/<version>/<kind>_types.go 
> project/controllers/<kind>_controller.go 

If the above not works then we could use as a workaround the core but in this way, IMO we need to find a way to fix the paths without set group="core" in the PROJECT file.

Reply:
You can close this one it is solved. I cloned your PR locally and re-check it. I like your solution here. It was a good way to solve it. 👍